tasks/copy_to_archive_adapter/bin/run_tests.sh

```diff
@@ -74,7 +74,7 @@ check_returncode $? "ERROR: Potential security or code issues found."
 echo "INFO: Running unit and coverage tests ..."
 
 # Currently just running unit tests until we fix/support large tests
-coverage run --source=copy_to_archive_adapter -m pytest
+coverage run --source=copy_to_archive_adapter/test -m pytest
```

Comment:
this was added to prevent conflict with the integration test

Reply:
If we move the integration tests to their own `integration_tests` folder, does that solve this? As-is, I think this will only display coverage results for unit tests.

Reply:
I did try moving the script under test/integration_tests/integration_test_copy_to_archive_adapter.py but got 
"Import error: No module named copy_to_archive_adapter" while running the bin/run_integration_tests.sh

I think that's what we want- bin/run_tests.sh to only show results for unit tests 
and bin/run_integration_tests.sh to only display results for integration tests

Reply:
Agreed. But we shouldn't change our coverage checks here. Try moving the integration tests to their own folder and running them with a `pytest` [command](https://docs.pytest.org/en/7.1.x/how-to/usage.html) instead of the `python integration_test_copy_to_archive_adapter.py`. Haven't run tests that way in a while, but try `pytest integration_test/`

Reply:
`pytest integration_tests/*`

ModuleNotFoundError: No module named 'copy_to_archive_adapter'

Reply:
This will make the coverage checks invalid. https://coverage.readthedocs.io/en/latest/source.html

Reply:
Use `coverage run --source=copy_to_archive_adapter -m pytest test/unit_tests` to only run tests in the unit_tests folder. Integration tests can then be placed in tests/integration_tests/test_copy_to_archive_adapter.py.`python -m pytest test/integration_tests` will then run the tests.

Reply:
aah I missed that part giving `Coverage.py warning: No data was collected. (no-data-collected)`. 
Reverted changes

Made changes according to your suggestion and worked fine!